packages/Testing/PHPUnit/Behavior/MovingFilesTrait.php

```diff
@@ -50,13 +50,13 @@ protected function assertFilesWereAdded(array $expectedAddedFileWithContents): v
             $expectedFilePathWithContent = $expectedAddedFileWithContents[$key];
 
             $this->assertSame(
-                $expectedFilePathWithContent->getFilePath(),
-                $addedFilePathWithContent->getFilePath()
+                realpath($expectedFilePathWithContent->getFilePath()),
+                realpath($addedFilePathWithContent->getFilePath())
```

Comment:
Now 2 method are used to compare.
That might be pain to debug later. How can we use just one method... E.g. getRealPath() or something

Reply:
this possibly related with https://github.com/rectorphp/rector-src/pull/127